submission/persistence-filesystem/src/main/kotlin/ac/uk/ebi/biostd/persistence/filesystem/fire/FireFilesService.kt

```diff
@@ -53,4 +37,16 @@ class FireFilesService(
         logger.info { "${sub.accNo} ${sub.owner} Finished persisting files of submission ${sub.accNo} on FIRE" }
         return submission
     }
+
+    override fun cleanSubmissionFiles(sub: ExtSubmission) {
+        fun cleanFile(file: FireFile, index: Int) {
+            logger.debug { "${sub.accNo}, ${sub.version} Cleaning file $index, path='${file.filePath}'" }
+            fireService.cleanFile(file)
+            logger.debug { "${sub.accNo}, ${sub.version} Cleaning file $index, path='${file.filePath}'" }
+        }
+
+        logger.info { "${sub.accNo} ${sub.owner} Cleaning Current submission Folder for ${sub.accNo}" }
+        serializationService.forEachFile(sub) { file, index -> if (file is FireFile) cleanFile(file, index) }
+        logger.info { "${sub.accNo} ${sub.owner} Cleaning Ftp Folder for ${sub.accNo}" }
```

Comment:
I think this log should say: Finished cleaning up submission folder for ${sub.accNo}